src/js/MainController.js

```diff
@@ -81,7 +81,7 @@ goog.require('ga_window_service');
         }),
         logo: false
       });
-
+      window.map = map;
```

Comment:
Is this really needed? Don't like wokring via global window object. Map should be passed around as parameter.

Reply:
probably added for testing purpose. I remove it.